mobile/swift-demo-social-login/node_server/index.js

```diff
@@ -0,0 +1,68 @@
+// server.js
+// where your node app starts
+
+// init project
+if (process.env.NODE_ENV !== 'production') require('dotenv').config()
+
+var express = require('express');
+var app = express();
+var CryptoJS = require('crypto-js');
+var DISQUS_SECRET = process.env.DISQUS_SECRET;
+var DISQUS_PUBLIC = process.env.DISQUS_PUBLIC;
+
+// http://expressjs.com/en/starter/static-files.html
+app.use(express.static('public'));
+
+// http://expressjs.com/en/starter/basic-routing.html
+app.get("/", function (request, response) {
+  response.sendFile(__dirname + '/views/index.html');
+});
+
+// listen for requests :)
+console.log("port is ", process.env.PORT);
```

Comment:
Can we remove this line since it repeats whats printed on the next line and instead print a message that includes the local server URL? (ie: http://localhost:5000/)